### ISSUE TITLE
Unbalanced vphantom escapes

### ISSUE THREAD

**Issue by bluteau** [retorquere/zotero-better-bibtex#1043]
- [x] I have verified in the Zotero addons screen that I have the latest release of BBT (https://github.com/retorquere/zotero-better-bibtex/releases/latest) and Zotero installed: 5.0.183 and 5.0.53 respectively
- [x] I am posting a single question, bug or feature request
- [x] I have included a descriptive subject of the problem
- [x] I am available for follow-up questions and testing
- [x] If I am posting an export or key-generation related bug report, I have included an error-report by right-clicking the problematic reference and submitting an BBT error report

# Bug classification

## Exporting references

Exporting to BibTeX. Report ID: 8C6MQIRD-euc

The exported .bib file worked flawlessly with BibTeX in my thesis LaTeX documents until somewhere between versions 5.0.176--179 (sorry for being slow on reporting, but I needed a break after submitting). BibTeX reports that the database file has an illegal ending, and a number of my entries cannot be found.

I have provided a MWE at https://drive.google.com/open?id=1ijuUWUL612qJgqXqyg5AiiaLR9BIAzLu

After running `pdflatex main.tex && bibtex main.aux` you should see the following output from bibtex:

```
This is BibTeX, Version 0.99d (TeX Live 2015/Debian)
The top-level auxiliary file: main.aux
The style file: unsrtnat.bst
Database file #1: phd_thesis.bib
Illegal end of database file---line 6426 of file phd_thesis.bib
 : 
 : 
(Error may have been on previous line)
I'm skipping whatever remains of this entry
Warning--I didn't find a database entry for "neu1996"
(There were 2 error messages)
```

So, the entry with citekey neu1996 is not being found in my .bib file (phd_thesis.bib) which was generated by Better BibTeX. I have a feeling that the problem arises because of how the entry titles are being handled (a quick diff with a previous version of my .bib file showed that bracketing {} of the titles has changed).

**Comment by blip-bloop** [comment]
:robot: this is your friendly neighborhood build bot announcing [test build 5.0.183.8009](https://github.com/retorquere/zotero-better-bibtex/releases/download/builds/zotero-better-bibtex-5.0.183.8009.xpi) ("fixes #1043").

**Comment by retorquere** [comment]
Try build 8009.

I also see that you use `url` fields -- note that bibtex does not support these by default, so you will need a custom style for them.

**Comment by retorquere** [comment]
You may also want to add this [postscript](https://retorque.re/zotero-better-bibtex/scripting/) to automatically detect `$`-surrounded LaTeX math:

```
if ((Translator.BetterBibTeX || Translator.BetterBibLaTeX) && this.has.title) {
  this.add({ name: 'title', value: item.title.replace(/(\$.*?\$)/g, '<pre>$1</pre>') });
}
```

**Comment by blip-bloop** [comment]
:robot: this is your friendly neighborhood build bot announcing [test build 5.0.183.8012](https://github.com/retorquere/zotero-better-bibtex/releases/download/builds/zotero-better-bibtex-5.0.183.8012.xpi) ("more test cases for #1043").

**Comment by retorquere** [comment]
I suspect the reason the earlier builds worked because you recently added `mies1973`; 5.0.176--179 would have had the same error as .183, but `mies1973` triggered the error.

**Comment by bluteau** [comment]
Thank you @retorquere !!! I have installed [test build 5.0.183.8012](https://github.com/retorquere/zotero-better-bibtex/releases/download/builds/zotero-better-bibtex-5.0.183.8012.xpi), and it seems to have resolved all of my issues. BibTeX finds all of my entries in the MWE I posted above and more importantly in my thesis document. I believe `mies1973` was a later addition to my database, so that was probably the culprit. 

The natbib package can handle the extra `url` field, but that is actually something I need to tidy up. I obviously don't want both a DOI and URL displayed for a reference, but I want at least one of them if the other is missing. My initial searches have shown that to be a non-trivial thing to achieve. Also, thanks for the postscript processing tip. I am not using titles in my current bibliography, but doubtless it will come in handy in the future. Cheers!
